typescript.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "commitMessageAction": "update",
+  "dependencyDashboard": true,
+  "dependencyDashboardAutoclose": true,
+  "enabledManagers": ["npm", "github-actions", "dockerfile"],
+  "extends": [
+    ":approveMajorUpdates",
+    ":automergeAll",
```

Comment:
maybe I misunderstand this, but doesn't `:automergeAll` preset [override](https://docs.renovatebot.com/key-concepts/presets/#how-to-use-presets) the `:approveMajorUpdates` preset?

Reply:
Only if there is a logical conflict, which depending on how you read it you can indeed see one 😅.
If I am not mistaken the result of this two presets will be:

* For non-major updates `automerge: true` and `dependencyDashboardApproval: false`.
* For major updates `automerge: true` and `dependencyDashboardApproval: true`.

Last one seems confusing but as the dashboard approval happens before the PR creation, it does not really conflict with `automerge`. AFAIK `dependencyDashboardApproval` kind of takes precedence.

Reply:
By the way, you can see how the major updates require approval with the same configuration here: https://github.com/ridedott/payments/issues/52.

Reply:
ah, I think I got it. thank you for the explanation and the example!